File: Java/AdvProg/Lab3/src/questions/Question3.java
package questions;
import java.util.Scanner;

public class Question3 {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		
		System.out.print("Number of names to process: ");
		int num = input.nextInt();
		String[] names = new String[num];

		
		for (int i = 1; i <= num; i++) {
			
			System.out.print("Enter the name " + i + ": ");
			String name = input.next();
			names[i - 1] = name; 
			
		}
		
		System.out.println("\nNames stored in Array: ");
		for (int i = 0; i < num; i++) {
			System.out.println("Element at index " + i + ": " + names[i]);
		}
		input.close();

	}

}

File: repos/Lab3CSharp/Lab3CSharp/Question3.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Lab3CSharp
{
    class Question3
    {
        public void qThree()
        {
            Console.Write("Number of names to process: ");
            int num = Convert.ToInt32(Console.ReadLine());
            string[] names = new string[num];
            for (int i = 0; i < num; i++)
            {
                Console.Write("Enter the name " + (i + 1) + ": ");
                names[i] = Console.ReadLine();
            }

            for (int i = 0; i < names.Length; i++)
            {
                Console.WriteLine("Elements at index " + i + ": " + names[i]);
            }
        }
        


    }
}
